zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -157,15 +107,15 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
         let callee_address = select::expr(
             is_callcode.expr() + is_delegatecall.expr(),
             current_callee_address.expr(),
-            code_address.expr(),
+            call_gadget.callee_address_expr(),
```

Comment:
Are `callee_address_expr` and `code_address` the same?

Reply:
From [the spec](https://github.com/privacy-scaling-explorations/zkevm-specs/blob/master/specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md#evm-behavior), the second arg should be callee address, not code address. I guess in the most of cases, it's a contract. So, the author name it `code_address`.

Reply:
Yeah, that makes sense to me.